contracts/lib/LibLiquidation.sol

```diff
@@ -44,7 +43,11 @@ library LibLiquidation {
         int256 totalBase
     ) internal pure returns (uint256) {
         int256 amountToEscrow = currentMargin - (minMargin.toInt256() - currentMargin);
-        int256 amountToEscrowProportional = PRBMathSD59x18.mul(amountToEscrow, PRBMathSD59x18.div(amount, totalBase));
+        int256 absoluteBase = PRBMathSD59x18.abs(totalBase);
+        int256 amountToEscrowProportional = PRBMathSD59x18.mul(
+            amountToEscrow,
+            PRBMathSD59x18.div(amount, absoluteBase)
+        );
```

Comment:
When `amountToEscrow` is negative (`currentMargin` is less than half of the `minimumMargin`),  this would have previously resulted in a positive `amountToEscrowProportional` when `amount` was negative. 

Since we are now forcing `PRBMathSD59x18.div(amount, absoluteBase)` to be positive, a negative `amountToEscrow` can never result in a positive `amountToEscrowProportional`.

Not sure if this would introduce an edge case for liquidating traders that are severely under margin?

Reply:
If `amountToEscrow` is negative, `amountToEscrowProportional` should be negative too. That way we know when a liquidator should not be escrowing anything.